src/main/java/org/apache/sling/feature/cpconverter/vltpkg/VaultPackageAssembler.java

```diff
@@ -88,30 +94,21 @@
             throw new IllegalStateException("Unable to create jcr root dir: " + jcrRootDirectory);
         }
 
-        PackageProperties packageProperties = vaultPackage.getProperties();
 
         Properties properties = new Properties();
+        Map<Object, Object> originalPackageProperties = vaultPackage.getMetaInf().getProperties();
```

Comment:
Is this the same as the old call above (vaultPackage.getProperties() ) ?

Reply:
It is the low-level API giving access to all package properties.